docs/content/AutoGPT/component agent/creating-components.md

```diff
@@ -0,0 +1,36 @@
+# Creating Components
+
```

Comment:
Why would I want to create a component? Give examples of how it works. 

ex: If you want to allow autogpt to send an email with gmail you can make a x or if you want to modify the prompt you can make a y or if you want to change behavior of w you can do z.